databrowser/src/domain/datasets/ui/toolBox/export/DownloadCard.vue

```diff
@@ -126,3 +152,9 @@ const download = () => {
   }
 };
 </script>
+
+<style scoped>
+.disabled{
+  opacity: 0.5;
+}
+</style>
```

Comment:
Please do not use custom style definitions if not absolutely necessary, it makes styling and style updates harder and, in the worst case, produces a dedicated css file and `link` section in the DOM like the one below (copied from production build of this PR):

```html
<link rel="stylesheet" crossorigin="" href="/assets/ColumnConfigurationNavigationGuardHandler-DEEvVKjm.css">
```

Use [tailwind custom styles instead](https://tailwindcss.com/docs/adding-custom-styles)